addons/account/data/invoice_action_data.xml

```diff
@@ -62,8 +62,7 @@ from ${object.company_id.name}.
             <field name="auto_delete" eval="True"/>
             <field name="body_html"><![CDATA[<html>
                 <head></head>
-                % set record = ctx.get('record')
-                % set company = record and record.company_id or user.company_id
+                % set company = ctx.get('company') or user.company_id
```

Comment:
should be `user.company_id.id` to be consistent

Reply:
well I was wondering which was best. It implies changing also the `/logo.png?company=${company.id}` below